src/ResourceManager/DataLakeStore/ChangeLog.md

```diff
@@ -17,6 +17,14 @@
     * Overview of change #1
         - Additional information about change #1
 -->
+## Current Release
+
+* Add debug functionality
+
+## Version 5.1.1
```

Comment:
I don't think you want to change the order of this

Reply:
@markcowl Addressed it.